webapp/src/app.tsx

```diff
@@ -4451,7 +4457,7 @@ document.addEventListener("DOMContentLoaded", () => {
     else if (isSandbox) workspace.setupWorkspace("mem");
     else if (pxt.winrt.isWinRT()) workspace.setupWorkspace("uwp");
     else if (pxt.BrowserUtils.isIpcRenderer()) workspace.setupWorkspace("idb");
-    else if (pxt.BrowserUtils.isLocalHost() || pxt.BrowserUtils.isPxtElectron()) workspace.setupWorkspace("fs");
+    else if (/*pxt.BrowserUtils.isLocalHost() ||*/ pxt.BrowserUtils.isPxtElectron()) workspace.setupWorkspace("fs");
```

Comment:
I'm curious why workspace is set to "fs" for localhost. Is this safe to remove? I'm seeing a network error locally when this is the case. It tries to GET /api/list, which doesn't exist in the backend.

Reply:
Projects are saved under the `projects` folder in the directory you run `pxt serve` in from with fs, instead of browser storage; very helpful for debugging shared scripts / etc

Reply:
Oh! I will revert this change.